docs/source/installation/index.rst

```diff
@@ -133,15 +133,19 @@ Optional configuration
 Verdi tab-completion
 ++++++++++++++++++++
 The ``verdi`` command line tool has many commands and options.
-To simplify its usage there is a way to enable tab-completion for it to your bash shell.
+To simplify its usage, there is a way to enable tab-completion for it to your bash shell.
 To do so, simply run the following command::
 
     $ verdi completioncommand
 
 and add the result to your bash config, e.g. ``.bashrc``, or if you are using a virtual environment, simply append it to the activation script.
-Alternatively, for both cases you can accomplish the same by simply by adding the following line to the ``.bashrc`` or activation script::
+Alternatively, you can accomplish the same by simply by adding the following line to the activation script::
 
-    eval "$(verdi completioncommand)
+    eval "$(verdi completioncommand)"
+
+.. note:: If you decide to use the above command (the evaluation of the ``verdi completioncommand``) you should not add it to the ``.bashrc``.
+          If, for any reason, it is necessary to have it in your ``.bashrc``, make sure that the virtual environment is activated before the execution
+          of that line.
 
```

Comment:
Yes, activating the VE in .bashrc cancels the main idea of having a VE (or at least one of them). And, maybe, this is enough as an argument to remove that line (I was also hesitating a bit when writing it - to activate the VE in the bashrc for this reason)

I understand that the other proposed solutions will work too but they are at the level of a "hack" (at least this is how they look to me from a first look).

I will remove the proposal to add 'eval "$(verdi completioncommand)"' in bashrc and I will only mention the activation script. In any case, the AiiDA installation guide assumes that VE is used.